Automattic/wp-calypso
# WooCommerce: Add the order list view

Fixes #13654 

<img width="1180" alt="screen shot 2017-06-20 at 9 47 49 am" src="https://user-images.githubusercontent.com/541093/27336461-a1d8291c-559d-11e7-94c0-5abb4f7f5c32.png">

This adds the basic orders list view. We've decided pagination will be part of v2, so right now you can only see a max of 100 orders. This particular PR just gets the basic display in, I'll add follow-ups for better currency display, and for the split status bar (in https://github.com/Automattic/wp-calypso/issues/13654#issuecomment-306613311).

To test:

- Visit `https://calypso.live/store/orders/$site`
- You should see the list of orders on your site
